Makefile

```diff
@@ -3,7 +3,11 @@
 all: proto
 
 proto:
-	protoc --go_out=plugins=grpc:../../../ -I. internal/proto/*.proto
+	docker build -t tilt-protogen -f Dockerfile.protogen .
+	docker rm tilt-protogen || exit 0
```

Comment:
fine as-is, but could change this to `docker rm -f tilt-protogen` to avoid the mildly distracting `Error: No such container: tilt-protogen` in the output

Reply:
sadly, that doesn't work.

```
$ docker rm -f tilt-protogen
Error: No such container: tilt-protogen
```

tho maybe we should complain to docker about it; it would be nice if it worked the same as `rm`

Reply:
oops, that's what I get for assuming it just ran `rm`